files/en-us/web/html/inline_elements/index.html

```diff
@@ -92,7 +92,7 @@ <h2 id="Conceptual_differences">Conceptual differences</h2>
  <dt>Content model</dt>
  <dd>Generally, inline elements may contain only data and other inline elements. You can't put block elements inside inline elements.</dd>
  <dt>Formatting</dt>
- <dd>By default, inline elements do not force a new line to begin in the document flow. Block elements, on the other hand, typically cause a line break to occur (although, as usual, this can be changed using CSS).</dd>
+ <dd>By default, inline elements do not force a new line to begin in the document flow. Block elements, on the other hand, typically cause a line break to occur (although, as usual, this can be changed using the CSS <a href="/en-US/docs/Web/CSS/display"><code>display</code></a> property).</dd>
```

Comment:
```suggestion
 <dd>By default, inline elements do not force a new line to begin in the document flow. Block elements, on the other hand, typically cause a line break to occur (although this can be changed using the CSS <a href="/en-US/docs/Web/CSS/display"><code>display</code></a> property).</dd>
```

I don't like assumptive phrases like this, so removing while we're here.